server/command/command.go

```diff
@@ -130,7 +130,7 @@ func getAutocompleteData(addTestCommands bool) *model.AutocompleteData {
 		testCreate.AddTextArgument("Name of the incident", "Incident name", "")
 		test.AddCommand(testCreate)
 
-		testData := model.NewAutocompleteData("bulk-data", "[ongoing] [ended] [begin] [end] [seed]", "Generate random test data in bulk")
+		testData := model.NewAutocompleteData("bulk-data", "[ongoing] [ended] [days] [seed]", "Generate random test data in bulk")
 		testData.AddTextArgument("An integer indicating how many ongoing incidents will be generated.", "Number of ongoing incidents", "")
 		testData.AddTextArgument("An integer indicating how many ended incidents will be generated.", "Number of ended incidents", "")
 		testData.AddTextArgument("Date in format 2020-01-31", "First possible creation date", "")
```

Comment:
We also need to update these descriptions. We need to modify this one as shown here and remove the next line

```suggestion
		testData.AddTextArgument("An integer n. The incidents generated will have a start date between n days ago and today.", "Range of days for the incident start date", "")
```

@justinegeffen may have a better wording for this.